resources/migrations/001_update_migrations.yaml

```diff
@@ -10469,6 +10469,37 @@ databaseChangeLog:
                   type: ${timestamp_type}
                   remarks: The timestamp at which a user was deactivated
 
+  - changeSet:
+      id: v53.2025-01-05T00:00:01
+      author: escherize
+      comment: set enable-embedding-sdk false when embedding-app-origins-sdk value exists
+      preConditions:
```

Comment:
Do you need `preConditions` here? Can it just be `preConditions: # not needed` if required by the linter?